### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by mrandri19**

### CODE REVIEW

**Comment by cmyr** [comment]
@mrandri19 what's the status of this? want to ping me when it's ready for a look?

**Comment by mrandri19** [comment]
@cmyr It's ready to be reviewed

File: druid-shell/src/gtk/dialog.rs [hunk 1]
```diff
@@ -48,25 +45,31 @@ pub(crate) fn get_file_dialog_path(
     ty: FileDialogType,
     options: FileDialogOptions,
 ) -> Result<OsString, Error> {
-    let dialog = match ty {
-        FileDialogType::Open => build_open_dialog(window, options),
-        FileDialogType::Save => build_save_dialog(window, options),
+    let (title, action) = match ty {
+        FileDialogType::Open => ("Open File", FileChooserAction::Open),
```
Comment (cmyr) [inline druid-shell/src/gtk/dialog.rs]: Out of scope for this patch, but we will need to figure out an API here that allows for localization; this means threading through the title from `druid`. You could add a `TODO` here, if you wanted.

File: druid-shell/src/gtk/mod.rs [hunk 2]
```diff
@@ -45,14 +46,31 @@ pub mod menu;
 pub mod util;
 pub mod win_main;
 
+macro_rules! clone {
```
Comment (cmyr) [inline druid-shell/src/gtk/mod.rs]: Can we add a comment to this, explaining what it's for? will be a point of confusion for anyone reading this code in the future.

File: druid-shell/src/gtk/mod.rs [hunk 3]
```diff
@@ -278,12 +285,11 @@ impl WindowBuilder {
...
 
                 Inhibit(true)
-            });
+            }));
         }
 
         {
-            let handle = handle.clone();
-            drawing_area.connect_scroll_event(move |_widget, scroll| {
+            drawing_area.connect_scroll_event(clone!(handle => move |_widget, scroll| {
```
Comment (cmyr) [inline druid-shell/src/gtk/mod.rs]: can we address this comment from the earlier review, and update one of the comments below?

https://github.com/xi-editor/druid/pull/120/files#r317243827

File: druid-shell/src/gtk/mod.rs [hunk 4]
```diff
@@ -46,6 +46,19 @@ pub mod menu;
...
+/// ```
+/// button.connect_clicked(clone!(handle => move |_| { ... }))
+/// ```
+/// is equivalent to:
+/// ```
+/// {
+///     let handle = handle.clone();
+///     button.connect_clicked(move |_| { ... })
+/// }
+/// ```
```
Comment (cmyr) [inline druid-shell/src/gtk/mod.rs]: awesome, thanks!

**Comment by cmyr** [comment]
This still crashes when running the `hello` example in `druid-shell`. Can you take a look at that?

**Comment by cmyr** [comment]
Okay the crash isn't happening for me right now, and I think that's enough to merge this.

Now we're going to have to merge in the new changes in druid, which will break our branch; I'll do the merge but leave the branch broken, and PRs can be made onto that branch to fix the new issues.

**Pull request merged**